internal/testing/fake/crypto.go

```diff
@@ -0,0 +1,388 @@
+package fake
+
+import (
+	"hash"
+
+	"go.dedis.ch/dela/crypto"
+	"go.dedis.ch/dela/serde"
+)
+
+// PublicKeyFactory is a fake implementation of a public key factory.
```

Comment:
In an ideal world we would keep the `- implements ...` convention, same for the functions. What do you think ?

Reply:
Yes, you're right, I wanted to do it but forgot to update the comments.